src/nxt_cgroup.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#include <nxt_main.h>
+#include <nxt_process.h>
+#include <nxt_fs.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <limits.h>
+
+
+static nxt_int_t nxt_cgroup_create(const nxt_task_t *task,
+    nxt_process_t *process);
+
+
+static nxt_int_t
```

Comment:
I prefer having `static` function definitions at the bottom (so that when I read a file, the first thing is the "important" extern stuff; only if I care enough to want to know implementation details I go to the bottom of the file).

Reply:
Comes from my natural tendency to write functions bottom up, generally doing away with the need to faff around with function prototypes...

Reply:
Function has been merged into nxt_cgroup_proc_add()